modules/pushapk_scriptworker/manifests/settings.pp

```diff
@@ -37,6 +37,19 @@
         verify_chain_of_trust    => true,
         verify_cot_signature     => true,
       },
+      'mobile-dep' => {
+        worker_group             => 'mobile-pushapk-dep-v1',
+        worker_type              => 'mobile-pushapk-dep-v1',
+        verbose_logging          => true,
+        taskcluster_client_id    => 'project/mobile/focus/releng/scriptworker/pushapk/dep',
+        taskcluster_access_token => secret('pushapk_scriptworker_taskcluster_access_token_mobile_dep'),
+        scope_prefix             => 'project:mobile:focus:releng:googleplay:product:',
```

Comment:
I'm tempted to say s/`focus`/`reference-browser` but let's see what the conclusions of the Slack thread on this is.

Reply:
Until https://github.com/mozilla-releng/pushapkscript/pull/54 gets merged, I can't add `project:mobile:reference-browser:...` to the `mobile-prod` env config.
My other option would be to implement this _and_ change `reference-browser` to have different pushapk scopes for dep and prod:
* `project:mobile:reference-browser:...` for dep
* `project:mobile:focus:...` for prod

But imho this looks more typo-y and confusing to future readers than using `focus` across the board (in the same way that we masquerade as `android-components`).
Either way, https://github.com/mozilla-releng/pushapkscript/pull/54 will allow us to do this without compromise

Reply:
Correct. Please keep it this way, for now :)